gallery/src/styles/NxCard/NxCardHorizontalExample.tsx

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { faShapes, faEdit, faTrash } from '@fortawesome/free-solid-svg-icons';
+
+import { NxFontAwesomeIcon, NxButton } from '@sonatype/react-shared-components';
+
+export default function NxCardHorizontalExample() {
+  return (
+    <div className="nx-card-container nx-card-container--column">
+      <section className="nx-card">
+        <header className="nx-card__header">
+          <h3 className="nx-h3">Header</h3>
+        </header>
+        <div className="nx-card__call-out">
+          <span className="nx-card__call-out-text">42</span>
```

Comment:
Outdated class

Reply:
> Outdated class. It's in a couple of places in the examples and also still mentioned in the documentation.

Here: https://github.com/sonatype/sonatype-react-shared-components/pull/170/commits/d24ea69de317ef47e3418aa579407c99fce3be42 and here: https://github.com/sonatype/sonatype-react-shared-components/pull/170/commits/29682dcab30f0d9ca6ffd4b490f5764de425794e